modules/experimental-layers/src/gpu-screen-grid-layer/gpu-screen-grid-layer.js

```diff
@@ -136,8 +136,8 @@ export default class GPUScreenGridLayer extends Layer {
   }
 
   calculateInstanceCounts(attribute, {numInstances}) {
-    // Empty method: Attribute manager requires an update method for each added attribute
-    // TODO: remove this when proper external buffer support is added to Attribute Manager.
+    const {aggregatedData} = this.state;
+    attribute.setExternalBuffer(aggregatedData.countsBuffer, numInstances);
```

Comment:
`attribute.update({buffer: aggregatedData.countsBuffer})` should work. `setExternalBuffer` is for attributeManager and I don't feel comfortable accessing it in layers.

Reply:
Updated.